src/ui/server/api.ts

```diff
@@ -202,6 +203,30 @@ const keystone = () => {
     }
   };
 
+  const getCompaniesHouseInformation = async (companiesHouseNumber: string) => {
```

Comment:
Wrap it in a `try-catch` for exceptional handling please.

Reply:
done